packages/lxd/src/storage/volume/attach.coffee.md

```diff
@@ -0,0 +1,106 @@
+
+# `nikita.lxc.storage.volume.attach`
+
+Attach a storage volume in the selected pool to an instance of LXD.
+
+## Output parameters
+
+* `$status`
+  True if the volume was attached.
+
+## Example
+
+```js
+const {$status} = await @lxc.storage.volume.attach({
+  pool = 'default',
+  name = 'test',
+  container = 'c1',
+  device = 'test'
+})
+console.info(`The volume was deleted: ${$status}`)
+```
+
+## Schema definitions
+
+    definitions =
+      config:
+        type: 'object'
+        properties:
+          'pool':
+            type: 'string'
+            description: 'Name of the storage pool containing the volume to attach.'
+          'name':
+            type: 'string'
+            description: 'Name of the storage volume to attach.'
+          'device':
+            type: 'string'
+            description: 'Name of the device as listed in the instance.'
+          'type':
+            enum: ["custom"]
+            default: "custom"
```

Comment:
Not sure about this bit. I would like to leave it as enum, so that it is easier to build on top of it.

Reply:
Fine with me